### ISSUE TITLE
Day1

### ISSUE THREAD

**Issue by sanshengshi626** [sanshengshi626/daka-pytorch#1]
pytorch基本概念：
在使用GPU时作为numpy库的替代品
提供最大程度的灵活度和速度的深度学习研究平台
代码：
import torch # arrays on GPU
import torch.autograd as autograd #build a computational graph
import torch.nn as nn ## neural net library
import torch.nn.functional as F ## most non-linearities are here
import torch.optim as optim # optimization package
# 2 matrices of size 2x3 into a 3d tensor 2x2x3
d=[[[1., 2.,3.],[4.,5.,6.]],[[7.,8.,9.],[11.,12.,13.]]]
d=torch.Tensor(d) # array from python list
print("shape of the tensor:",d.size())

# the first index is the depth
z=d[0]+d[1]
print("adding up the two matrices of the 3d tensor:",z)
shape of the tensor: torch.Size([2, 2, 3])
adding up the two matrices of the 3d tensor:
  8  10  12
 15  17  19
[torch.FloatTensor of size 2x3]
# a heavily used operation is reshaping of tensors using .view()
print(d.view(2,-1)) #-1 makes torch infer the second dim
  1   2   3   4   5   6
  7   8   9  11  12  13
[torch.FloatTensor of size 2x6]
